From 5247a22a650d7a74eb8ecf1a044b34c56bf57500 Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@freescale.com>
Date: Tue, 6 Jan 2015 17:28:28 +0800
Subject: [PATCH 0299/1594] MLK-10075: ARM: clk-imx6q: refine esai_ipg's
 parent

commit 36e3392de98dd2e5659ab66356cd6be81526ef2d from
git://git.freescale.com/imx/linux-2.6-imx.git

esai_ipg clock's parent is ahb, not ipg.

Signed-off-by: Shengjiu Wang <shengjiu.wang@freescale.com>
---
 arch/arm/mach-imx/clk-imx6q.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-imx/clk-imx6q.c b/arch/arm/mach-imx/clk-imx6q.c
index 94a95ef..b66cf51 100644
--- a/arch/arm/mach-imx/clk-imx6q.c
+++ b/arch/arm/mach-imx/clk-imx6q.c
@@ -1,5 +1,5 @@
 /*
- * Copyright 2011-2013 Freescale Semiconductor, Inc.
+ * Copyright 2011-2015 Freescale Semiconductor, Inc.
  * Copyright 2011 Linaro Ltd.
  *
  * The code contained herein is licensed under the GNU General Public
-- 
1.7.5.4

